assets/src/components/activities/multi_input/MultiInputDelivery.tsx

```diff
@@ -120,6 +120,22 @@ export const MultiInputComponent: React.FC = () => {
     {},
   );
 
+  const inputValue = (input: MultiInput) => {
+    const studentInput = uiState.partState[input.partId].studentInput;
```

Comment:
The old version has a null guard:
uiState.partState[input.partId]?.studentInput || []

Might that be null or undefined?